IOTConnectSDK/iotconnect-sdk_2_1/src/main/java/com/iotconnectsdk/SDKClient.kt

```diff
@@ -202,5 +228,49 @@ class SDKClient(
         sdkClientManager?.sendData(jsonData)
     }
 
+
+    /**
+     *Get child device from IOT connect portal
+     *https://docs.iotconnect.io/iotconnect/resources/device-message-2-1-2/device-identity-messages/#devices
+     *
```

Comment:
@sw-tt-tusharojha @sw-tt-rajkiranvank  add more documentation

Reply:
@sw-ot-deveshmevada 
Comment added for the function